docs/lavamoat-policy-review-process.md

```diff
@@ -0,0 +1,29 @@
+# LavaMoat Policy Review process in metamask-extension
+
+When there's a need to change policy (because of new or updated packages that require a different set of capabilities), please follow these steps:
+
+> In the initial soft-launch of the process the approval from policy reviewers is not mandatory, but it will be in the near future.
+
+### Engineer on the dev team:
+   1. Notice the `validate-lavamoat-policy-*` circleci job fail because dependency updates or changes need a policy update.
```

Comment:
We recently migrated this check to GitHub Actions, and the name changed slightly:
```suggestion
   1. Notice the `Validate lavamoat policy*` PR status check fail because dependency updates or changes need a policy update.
```